modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -362,15 +362,19 @@ export default function _JournalPortlet({
 
 						if (url.searchParams.has(key)) {
 							articleId = url.searchParams.get(key);
+							document.getElementById(
+								`${namespace}articleId`
+							).value = articleId;
 						}
 					}
-
-					lockHolder.lock?.unlock();
 				}
 			})
 			.catch((error) => {
 				console.error(error);
 				lockHolder.lock?.unlock();
```

Comment:
As we added a finally, I would say this is not necessary anymore

Reply:
cc:// @NemethNorbert

Reply:
Yes we can remove`lockHolder.lock?.unlock();` from the catch block as well